Data/models/Model.xcdatamodeld/Model11.xcdatamodel/contents

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
what's new here? `RSSFeedSource` entity only?

Reply:
Yes, `RSSFeedSource` is added, nothing else has changed